AutoCollection/ServerRequests.ts

```diff
@@ -93,18 +101,25 @@ class AutoCollectServerRequests {
             return;
         }
 
+        // Add the target ikey hash to the response headers, if not already provided.
+        if (client.config && client.config.instrumentationKeyHash &&
+            response.setHeader && !response.getHeader(RequestResponseHeaders.targetInstrumentationKeyHeader)) {
+                response.setHeader(RequestResponseHeaders.targetInstrumentationKeyHeader,
+                    client.config.instrumentationKeyHash);
+        }
```

Comment:
niggle: Exact same code as above. Worth making a private function to avoid duplication?